services/src/main/java/org/openlearn/service/StorageService.java

```
@@ -68,39 +94,48 @@ public FileInformation store(final MultipartFile file, Long assignmentId, Long p
 		}
 
 
-//		Course course = new Course();//courseRepository.findOne(courseId);
```

It concerns me that the try/catch above is just swallowing exceptions.

---

Yup. That's a holdover from the first implementation of file uploads. It caused problems for me earlier when I was accidentally using Taylor's bucket and had the store process returning successfully even though I couldn't actually upload to that bucket. I'm in the works of refactoring that catch to log and throw again (and only catch amazon exceptions).